### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by vishalvishw10**
I have corrected some spelling errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ For front-end designing I have used Twitter's front-end library Bootstrap4.
 
 ### Features of the quiz:
 
-* Every question is multiple choice question.
+* Every question is multiple-choice questions.
```
Comment (hugovk) [inline README.md:24]: This makes it worse

Comment (akashgiricse) [reply README.md:24]: Correct.

Comment (akashgiricse) [reply README.md:24]: @vishalvishw10 Please fix and create another PR. Closing this PR.

Comment (hugovk) [reply README.md:24]: @akashgiricse Please could you add the `invalid` label too, to make sure this spam doesn't count towards a free Hacktoberfest tshirt?

(Edit: Actually, some of the changes were okay.)

Comment (akashgiricse) [reply README.md:24]: Added. Thanks.

**Pull request closed without merging**